docs/admin/tutorial.md

```diff
@@ -1,53 +1,81 @@
 # Tutorial
 
-This tutorial will show you how to use bpfd.
+This tutorial will show you how to use `bpfd`.
+There are several ways to launch and interact with `bpfd` and `bpfctl`:
 
-## Prerequisites
+* Basic Mode - Run `bpfd` in foreground or background and straight from build directory.
+  No `bpfd` or `bpfctl` users are created so always need `sudo` when executing `bptctl` commands.
+  See [Basic Mode](#basic-mode).
+* `bpfctl` without `sudo` - Run `bpfd` in foreground or background and straight from
+  build directory, but also create the `bpfd` or `bpfctl` users and user groups.
+  So if `bptctl` user group is added to a given user, `sudo` is no longer needed when executing `bptctl` commands.
+  See [`bpfctl` without `sudo`](#bpfctl-without-sudo).
+* Systemd Service - Run "bpfd" as a systemd service.
+  See [Systemd Service](#systemd-service).
+
+## Basic Mode
+
+### Step 0: Prerequisites
 
 This tutorial uses examples from the [xdp-tutorial](https://github.com/xdp-project/xdp-tutorial).
 You will need to check out the git repository and compile the examples.
 
-## Step 1: Build bpfd
+### Step 1: Build `bpfd`
+
+Perform the following steps to build `bpfd`.
 
-``` console
+```console
 cd $HOME/src/bpfd/
 cargo xtask build-ebpf --libbpf-dir $HOME/src/libbpf
 cargo build
 ```
 
-## Step 2: Start bpfd
+### Step 2: Setup `bpfd` environment
+
+`bpfd` uses mTLS for mutual authentication.
+In order to run and interact with `bpfd`, certificates must be created.
+Scripts to create self-sign certificates can be used in a development environment.
```

Comment:
```suggestion
Scripts to create a local certificate authority which is used to sign certs for bpfd and bpfctl can be used in a development environment.
```

I think is the right terminology, since "technically" they aren't self signed certs

Reply:
ack